index.html

```diff
@@ -116,9 +116,9 @@ <h2 class="worktitile"> MY RECENT WORK</h2>
 
 
                 <li class="workCard" id="first">
-
-                    <img src="Images/projectOne.png" alt="projectone">
-
+                    <div class="image_container">                
```

Comment:
- [ ] Your website is online :+1: . But as the project reviewer mentioned, there are some issues you need to address, you need to adjust your mobile version online. Please add some margin to all your single project card because by sticking to the Figma template we help you to have a good User Experience empathizing with users to ensure their journey through your website is both functional and pleasurable. :smile: